src/main/java/com/atlassian/jira/cloud/jenkins/buildinfo/client/model/Builds.java

```diff
@@ -3,24 +3,31 @@
 import com.atlassian.jira.cloud.jenkins.common.client.JiraRequest;
 import com.atlassian.jira.cloud.jenkins.common.client.model.Properties;
 import com.atlassian.jira.cloud.jenkins.common.client.model.ProviderMetadata;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
 
 import java.util.Collections;
 import java.util.List;
 
 /** This represents the payload for the API request to submit list of builds */
 public class Builds implements JiraRequest {
-    private List<JiraBuildInfo> builds;
+    @JsonIgnore private JiraBuildInfo build;
```

Comment:
The `JiraRequest` object is subject to JSON serialization. Since the Jira API requires a list of builds, this object has a list of builds. But we're ALWAYS sending one build at a time, so we had to do ugly things like `getBuilds().get(0)`. 

I replaced the `builds` property with a `build` property so we don't have to do these ugly things any more and added `getBuilds()` so that the JSON serialization still renders a `builds` field to satisfy the Jira API.